mne/viz/topomap.py

```diff
@@ -541,7 +543,20 @@ def set_values(self, v):
         # Eventually we could also do set_values with this class if we want,
         # see scipy/interpolate/rbf.py, especially the self.nodes one-liner.
         from scipy.interpolate import CloughTocher2DInterpolator
-        v = np.concatenate((v, np.zeros(self.n_extra)))
+
+        if isinstance(self.border, Integral):
+            v_extra = np.ones(self.n_extra) * self.border
+        elif isinstance(self.border, str) and self.border == 'mean':
```

Comment:
```
_validate_type(self.border, ('numeric', str), 'border')
if isinstance(self.border, str):
    if self.border != 'auto':
        raise ValueError('border must be numeric or "auto", got %r' % (self.border,))
else:
    border = np.full(self.n_extra, border, dtype=float)
```

Reply:
ah, yes, forgot about the safety checks :)

Reply:
and thanks for the `np.full()` suggestion, I wasn't aware of it!